Sources/Sentry/SentryHub.m

```diff
@@ -77,6 +86,114 @@ - (instancetype)initWithClient:(nullable SentryClient *)client
     return self;
 }
 
++ (void)enrichScope:(SentryScope *)scope crashWrapper:(SentryCrashWrapper *)crashWrapper
```

Comment:
`m`: The hub is already huge. I would prefer to have this code in an extra file. It can be a helper method somewhere.